src/Resources/config/validator.xml

```diff
@@ -9,6 +9,9 @@
             <tag name="validator.constraint_validator" />
             <argument type="service" id="libphonenumber\PhoneNumberUtil" />
             <argument>%misd_phone_number.validator.default_region%</argument>
+            <call method="setPropertyAccessor">
+                <argument type="service" id="property_accessor" on-invalid="ignore"/>
+            </call>
```

Comment:
Why not inject this service in the constructor?

Reply:
To avoid the complexity of the construct. I would have it as first or second parameter but it will create a BC.
I can change if you have a suggestion :)

Reply:
You're right, let's avoid BC break! :ok_hand: